citations-tool/tool/src/webapp/js/edit_nested_citations.js

```diff
@@ -51,6 +57,7 @@
             onClick( $('#' + toggleId).get(0), toggleEditor );
             onClick( $('#' + removeDivId).get(0), removeSection );
             onClick( $('#' + addSubsectionButtonId).get(0), addSubsection );
+            onClick( $('#' + addDescriptionH1ButtonId).get(0), addSubsection );
```

Comment:
The `addSubsection` function is getting long and harder to read, it may be worth splitting up in the future.